### PR TITLE
Test: Fix CIDR/L4 flakes

### PR DESCRIPTION
**Pull request by eloycoto**
Due the connectivity test to CloudFlare, time to time this test was not
reliable at all due internet connectivity issues. With this change if
the connection is failing more than one time will fail.

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

Fixes: #5745

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/6241)

### CODE REVIEW

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

File: test/runtime/Policies.go [hunk 1]
```diff
@@ -1379,7 +1381,44 @@ var _ = Describe("RuntimePolicies", func() {
 				}]
 			}]`, helpers.App1, cloudFlare)
 
-			testCIDRL4Policy(policy, cloudFlare, "https")
+			_, err := vm.PolicyRenderAndImport(policy)
+			Expect(err).To(BeNil(), "Unable to import policy")
+
+			httpd2, err := vm.ContainerInspectNet(helpers.Httpd2)
+			ExpectWithOffset(1, err).Should(BeNil(),
```
Comment (ianvernon) [inline test/runtime/Policies.go]: Change all `ExpectWithOffset` in this diff to `Expect`.

**Review by ianvernon — COMMENTED** [review]
LGTM apart from `ExpectWithOffset` changes as I commented.

File: test/runtime/Policies.go [hunk 2]
```diff
@@ -1360,6 +1360,8 @@ var _ = Describe("RuntimePolicies", func() {
 
 		It("Tests egress with CIDR+L4 policy to external https service", func() {
 			cloudFlare := "1.1.1.1"
+			proto := "https"
```
Comment (aanm) [inline test/runtime/Policies.go:1363]: why is this variable here? What does it do?

**Review by aanm — CHANGES_REQUESTED** [review]

File: test/runtime/Policies.go (hunk 2 above)
Comment (eloycoto) [reply test/runtime/Policies.go:1363]: Because is used in a lot of curl fails and the test policy is restricted to port 443. 

Regards

**Comment by eloycoto** [comment]
test-me-please

**Comment by eloycoto** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]
LGTM!

**Comment by ianvernon** [comment]
@eloycoto does this fix any of the issues in the CI Project? If so can you mark them as having been fixed by this PR?

**Comment by eloycoto** [comment]
@ianvernon is already in the commit message ;-)

**Comment by ianvernon** [comment]
@eloycoto thanks! I added it in the PR description as well.

File: test/runtime/Policies.go (hunk 2 above)
Comment (ianvernon) [reply test/runtime/Policies.go:1363]: @aanm can you PTAL at @eloycoto 's comment?

Comment (aanm) [reply test/runtime/Policies.go:1363]: But why does it need to be a variable and not use `https` directly?

Comment (eloycoto) [reply test/runtime/Policies.go:1363]: @aanm because it is used four times in the test, and it's the way that the test was before.

**Review by aanm — APPROVED** [review]

**Pull request merged**